extend/browsing-downloading-from-extend.md

```diff
@@ -10,66 +10,78 @@ categories: [extend]
+- Translations
 
-- Language Packs
+- Graphics
 
 Each main category has sub-categories, each with filtering options.
 
 
 ### Browsing a category
 
-[![Browsing and Filtering](/assets/images/mods/downloaders1.png)](/assets/images/mods/downloaders1.png)
+When you visit a category page, here's how it looks like:
+
+[![Browsing and Filtering](/assets/images/extend/downloaders1.png)](/assets/images/extend/downloaders1.png)
+
+For each projects you can view the cover and this informations:
+
+- Title
+
+- Small Description
 
-Placing the mouse over one of the project boxes quickly allows you to download the latest **stable** build and view how many times the project has been recommended.
+- Stars
 
-Clicking the project names directs you towards the project page, on which you can see detailed information: full description, previews, license, suggestions, bugs, etc.
+- Downloads
 
+- Last Update
+
+There is also a download button that allows you to download the latest **stable** build.
+
+Clicking the project names or the Details button directs you towards the project page, on which you can see detailed information: full description, previews, license, suggestions, bugs, etc.
+
+On the right side you can select a subcategory.
 
 ### Viewing a Project
 
 When you visit a project page, here's how it looks like:
 
-[![Project Page](/assets/images/mods/downloaders2.png)](/assets/images/mods/downloaders2.png)
+[![Project Page](/assets/images/extend/downloaders2.png)](/assets/images/extend/downloaders2.png)
 
-You can expand and collapse the Information, Previews and License sections to view/hide their content.
-
-On the right side, you can see the following options:
+Under description there are 4 buttons:
 
 - Download (download page)
 
-- License (expands the license section)
+- Star (allows you to recommend projects)
 
-- Change Logs (page where the changes for each build are displayed)
+- Get Support (either the support link entered by the project author or a link to the support forum on the MyBB Community)
 
 - Report (where you can report a project)
 
-- Recommend (allows you to recommend projects)
+Under this buttons there is a tab with the following links:
+
+- Details (where you can see project details)
 
-- Support (either the support link entered by the project author or a link to the support forum on the MyBB Community)
+- Change Log (page where the changes for each build are displayed)
 
 - Bug Tracker (you will see this option if it's been enabled by the project author)
 
 - Suggestions (you will see this option if it's been enabled by the project author)
 
-- Translations (translations posted by collaborators or the project owner)
-
+- License (where you can see the license of the project)
 
-If the owner of the project set a PayPal e-mail for donations, you will see a Donate button on the right as well.
+On the right side, you can see the latest build and a Donate button if the owner of the project set a PayPal e-mail for donations.
```

Comment:
Either you use present tense (sets) or present perfect (has set). I'd go with the latter.